Tools/TestWebKitAPI/Tests/WebKitCocoa/ScreenTime.mm

```diff
@@ -683,6 +683,22 @@ - (void)userContentController:(WKUserContentController *)userContentController d
     EXPECT_FALSE(suppressUsageRecording);
 }
 
+TEST(ScreenTime, DoNotCreateControllerWhenWebViewAlwaysOffscreen)
```

Comment:
What behavior do you want for web view that becomes off screen at some indeterminate point? You would still have an `STWebPageController` around, is that ok?

Reply:
At first, I thought it would be okay to keep it around since donations are suppressed in off screen web views anyway. But on second thought, the XPC connection would be unnecessarily kept alive, which is a kind of memory leak.

Due to that, we should uninstall the controller if the web view goes offscreen.